love/src/Config.js

```diff
@@ -127,6 +127,8 @@ export const severityEnum = {
   critical: 4,
 };
 
+export const ALARM_SOUND_THROTLING_TIME = 6000;
```

Comment:
Please add a comment explaining the unit. I guess it is ms but I could be wrong.